.jshintrc

```diff
@@ -1,3 +1,6 @@
 {
-	"indent": "tab"
+	"indent": "tab",
+	"undef": true,
+	"unused": true,
+	"predef": [ "require", "module", "process", "console"]
```

Comment:
Should probably be using `"node": true` instead of `predef` (see http://jshint.com/docs/options/#node)